src/routes/persons.ts

```diff
@@ -66,6 +67,23 @@ export const createPerson = async (req, res) => {
   return createdPerson;
 };
 
+export const deletePerson = async (req, res) => {
+  const { id: personId } = req.params;
+  try {
+    const isPersonDeleted = await removePerson(personId);
+    return res.status(200).send(isPersonDeleted);
+  } catch (err) {
+    return res.status(500).send({
+      errors: [
+        {
+          id: "0a5ec2ea-6772-11e9-a656-02420a868404",
```

Comment:
🔧 I presume this is hardcoded.

```suggestion
          id: personId,
```

Reply:
actually, i don't know what the id refers to but I found it already used in different places so I thought it referred to the  id of the error itself you can check the `showPerson` in the following path `src/routes/persons.ts`

Reply:
According to the response of `showPerson` this id seems in fact to be the error id 👍🏼 
🔧  @EslaM-M it would be worth it to store it in a constant with an explicit name now that we're using it 3 times in this file, what do you think?

Reply:
@Campalo thanks for the suggestion here is the commit for this suggestion
[e3513aa](https://github.com/kontist/mock-solaris/pull/169/commits/e3513aa747c439c3409ac6ef8976e943659e5e9d)